devops/Dockerfile

```
@@ -29,8 +29,10 @@ COPY devops/nginx/nginx.conf /etc/nginx/nginx.conf
 COPY devops/nginx/sites-enabled/default.conf /etc/nginx/sites-enabled/default.conf
 COPY devops/supervisord.conf /etc/supervisord.conf
 
-RUN echo "upload_max_filesize=200M" > /usr/local/etc/php/conf.d/uploads.ini
-RUN echo "post_max_size=200M" >> /usr/local/etc/php/conf.d/uploads.ini
+RUN echo "upload_max_filesize=500M" > /usr/local/etc/php/conf.d/uploads.ini
+RUN echo "post_max_size=500M" >> /usr/local/etc/php/conf.d/uploads.ini
+RUN echo "memory_limit=512M" >> /usr/local/etc/php/conf.d/uploads.ini
```

Consider combining all PHP configuration directives into a single `RUN` command to reduce the number of layers in the Docker image:
```suggestion
RUN echo "upload_max_filesize=500M" > /usr/local/etc/php/conf.d/uploads.ini \
    && echo "post_max_size=500M" >> /usr/local/etc/php/conf.d/uploads.ini \
    && echo "memory_limit=512M" >> /usr/local/etc/php/conf.d/uploads.ini \
    && echo "max_execution_time=300" >> /usr/local/etc/php/conf.d/uploads.ini
```